python/tvm/relay/frontend/onnx.py

```diff
@@ -3935,6 +3935,58 @@ def try_resolve_to_const(x, dtype_override=None):
         return y
 
 
+class MatMulInteger(OnnxOpConverter):
+    """Operator converter for MatMulInteger."""
+
+    @classmethod
+    def _impl_v10(cls, inputs, attr, params):
+        a = inputs[0]
```

Comment:
Hmm so pretty sure you can just do `relay.nn.matmul(..., out_dtype='int32')`. You don't need all of this QLinearMatMul stuff to handle accumulation without overflow imo.

Reply:
If scale is always 1 and zero point is always 0, I think that's true.

Reply:
My impression was that `MatMulInteger` exactly corresponds to our `qnn.dense`. We can use `QLinearMatMul` converter but we want to skip requantize.

Reply:
https://github.com/onnx/onnx/blob/main/docs/Operators.md#matmulinteger <-- I was fixated on the line where it said it was identical to numpy's. It appears the zero points are not fixed always so using QLinearMatMul is probably best choice.

Reply:
modified `QLinearMatMul` converter to skip requantize in the MatMulInteger case